TrafficCapture/captureOffloader/build.gradle

```diff
@@ -31,6 +31,8 @@ dependencies {
     testImplementation group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.20.0'
     testImplementation group: 'org.apache.logging.log4j', name: 'log4j-slf4j2-impl', version: '2.20.0'
     testImplementation group: 'org.slf4j', name: 'slf4j-api', version: '2.0.7'
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '5.10.0'
+    testImplementation group: 'org.mockito', name: 'mockito-junit-jupiter', version: '5.10.0'
```

Comment:
Adding mockito is a not net positive for this PR.  Mocks should be a last resort & for functional testing, we don't need it.  I see testing without mocks as an important design test - it makes the codebase more flexible to be repurposed later on.

Reply:
Removed in recent revision